src/App/LoadFigure.purs

```diff
@@ -55,22 +53,36 @@ loadSpec filename = do
       Left err -> error ("Json fetching failed with " <> printError err)
       Right response -> pure $ response.body
 
-loadFigureFromJson :: Json -> Effect Unit
-loadFigureFromJson json = runAffs_ (uncurry drawFig)
-   [ case decodeJson json :: Either JsonDecodeError JsonSpec of
-        Left err -> error ("JSON decoding failed with " <> show err)
-        Right spec -> ("fig" × _) <$> runWebT (FileCxt { fluidSrcPaths }) (loadFig figSpec)
-           where
-           figSpec@{ fluidSrcPaths } = figSpecFromJson spec
-   ]
+--loadFigure -- specFilename (String) & srcFilename (String)
```

Comment:
These comments don't add any information that isn't given in the type signature and first line of the function definition -- I think they can just be removed.